src/tools/test_trajectory.cpp

```diff
@@ -158,6 +158,10 @@ class TestTrajectory
   // A shared node handle
   ros::NodeHandle nh_private_;
 
+  // A string containing the 'trajectory_controller' parameter value
+  std::string trajectory_controller;
+
+
```

Comment:
A tiny little nitpick - delete these whitespace lines

Reply:
Merged your patch over in my repo.  Thanks!